docs/requirements.txt

```diff
@@ -1,13 +1,13 @@
--i https://pypi.python.org/simple
--f https://download.pytorch.org/whl/torch_stable.html
+# docs requirements
+torch
+torchvision
```

Comment:
Don't you want specific versions of `torch` and `torchvision`?

Reply:
maybe a lower bound >=1.6

Reply:
It should be the same that you require for installation. Otherwise you might get a different behavior.

Reply:
then >= 1.6